lib/house.rb

```diff
@@ -0,0 +1,47 @@
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
```

Comment:
be careful here-- the interaction pattern asks price to be an integer, so this should be 
`@price = price.to_i`